pynest/examples/sudoku/helpers.py

```diff
@@ -233,73 +207,119 @@ def plot_field(puzzle, solution, with_color=False):
         puzzle (np.array): array of shape (9,9) that represents the puzzle
         that is being solved. See get_puzzle()
         solution (np.array): array of shape (9,9) representing the solution.
+        ax (plt.Axes): Axes object on which to draw the field.
         with_color (bool, optional): if True, green and red are used to
         indicate which parts of the solution are valid and which are not.
         Otherwise, only black and white are used. Defaults to False.
 
     Returns:
         PIL.Image: A visual representation of the Sudoku solution.
```

Comment:
@krachbumm3nte: can you please reply to this?

Reply:
Apologies, I was Ill over the last week. It turns out that this function does not need to return anything, since the axis object is passed as an argument and modified in place. I also realized, that I only tested the new implementation with the plot_progress.py script, not noticing that it is also used in sudoku_solver.py. I changed the call there aswell.